pennylane/resource/first_quantization.py

```diff
@@ -83,23 +89,47 @@ def __init__(
         self,
         n,
         eta,
-        omega,
+        omega=None,
         error=0.0016,
         charge=0,
         br=7,
+        vectors=None,
     ):
         self.n = n
         self.eta = eta
         self.omega = omega
         self.error = error
         self.charge = charge
         self.br = br
+        self.vectors = vectors
+        self.cubic = True
+
+        if omega is None and vectors is None:
+            raise ValueError("The lattice vectors must be provided.")
```

Comment:
```suggestion
            raise ValueError("The lattice vectors or unit cell volume (omega) must be provided.")
```

Reply:
I intentionally included only lattice vectors because we want to deprecate unit cell volume soon and don't want to advertise it anymore.